doc/ext/imgconverter.rst

```diff
@@ -0,0 +1,30 @@
+.. highlight:: rest
+
+:mod:`sphinx.ext.imgconverter` -- Convert images to appropriate format for builders
```

Comment:
As a result of discussion in hack-day, we agreed with current name: `sphinx.ext.imgconverter`.